database/schema/upgrade_scripts/078-system-platform-uuid.sql

```diff
@@ -0,0 +1,99 @@
+-- Vanish non-uuid systems from DB
+CREATE OR REPLACE FUNCTION try_cast_uuid(uuid_in TEXT) 
+  RETURNS uuid AS
+$$
+  BEGIN
+    RETURN uuid_in::UUID;
+  EXCEPTION WHEN invalid_text_representation THEN 
+    RETURN NULL;
+  END;
+$$ LANGUAGE 'plpgsql';
+
+SELECT delete_system(inventory_id) FROM system_platform WHERE try_cast_uuid(inventory_id) IS NULL;
+
+ALTER TABLE system_platform ADD COLUMN inventory_id_new UUID;
+CREATE INDEX ON system_platform(inventory_id_new);
```

Comment:
creating index is not needed because adding UNIQUE on line 55 creates unique index automatically